src/librustc_middle/ty/sty.rs

```diff
@@ -1793,7 +1791,11 @@ impl<'tcx> TyS<'tcx> {
 
     #[inline]
     pub fn is_phantom_data(&self) -> bool {
-        if let Adt(def, _) = self.kind { def.is_phantom_data() } else { false }
+        if let Adt(def, _) = self.kind {
+            def.is_phantom_data()
+        } else {
+            false
+        }
```

Comment:
This (and the other similar change) don't look right, can you try `./x.py fmt`? You may have formatted them with a newer version of `rustfmt`.

Reply:
Good catch! Fixed now.